modules/vaos/app/services/ccra/configuration.rb

```diff
@@ -52,7 +52,6 @@ def connection
         end
 
         conn.response :betamocks if mock_enabled?
-        conn.response :snakecase
```

Comment:
I believe you mentioned this is the setting that also symbolizes? Would it be better to have consistency across our configurations? 

Eps service config: https://github.com/department-of-veterans-affairs/vets-api/blob/89befdca9c9e6085fe5bc479bca81321fd3b5ec0/modules/vaos/app/services/eps/configuration.rb#L35

vaos service config: https://github.com/department-of-veterans-affairs/vets-api/blob/89befdca9c9e6085fe5bc479bca81321fd3b5ec0/modules/vaos/app/services/vaos/configuration.rb#L36